src/pages/scheduler/components/CourseCard.tsx

```diff
@@ -117,13 +119,15 @@ export function CourseCard({
               size="xs"
               onClick={onDelete}
             />
-            <IconButton
-              aria-label="More information"
-              icon={showSections ? <ChevronUpIcon boxSize="1.5em" /> : <ChevronDownIcon boxSize="1.5em" />}
-              colorScheme="blue"
-              size="xs"
-              onClick={onShowSections}
-            />
+            {showSectionsToggle && (
+              <IconButton
+                aria-label="More information"
+                icon={showSections ? <ChevronUpIcon boxSize="1.5em" /> : <ChevronDownIcon boxSize="1.5em" />}
+                colorScheme="blue"
+                size="xs"
+                onClick={onShowSections}
+              />
+            )}
```

Comment:
I also realized you chose to hide it intentionally. This is why I shouldn't review PRs in the morning :facepalm:

Reply:
I think we should do something that more clearly conveys that a course isn't offered. Disabling & not showing the button might not be 100% clear and could leave people wondering why they can't see sections and being unable to click the chevron button could appear "broken".

Curious to hear thoughts but how about something like

- Card is greyed out
- Chevron button & check box aren't rendered
- The course title is replaced with something like `Course not offered for Spring 2022`

Reply:
I think we should do something that more clearly conveys that a course isn't offered. Disabling & not showing the button might not be 100% clear and could leave people wondering why they can't see sections and being unable to click the chevron button could appear "broken".

Curious to hear thoughts but how about something like

- Card is greyed out
- Chevron button & check box aren't rendered
- The course title is replaced with something like `Course not offered for Spring 2022`

Reply:
I like the idea of removing the checkbox toggle to show the course on the left side. 

I don't think greying out the card is the best option and neither is replacing the title.

A potential idea to expand on would be to use the chevron toggle to display a message that there aren't any sections being offered and to check with your department instead of showing sections. (However, this might also cause some confusion).

---

My reasoning behind this PR is that I am taking CSC 490 next term, but it doesn't have sections. But I still want it visible while I am planning my schedule.

<img width="1343" alt="157901484-7aef6d14-7339-4d4d-87d3-55852663fbdb" src="https://user-images.githubusercontent.com/10472448/157984481-ce313d0b-32c6-4db8-be34-9bfb405f3368.png">

Reply:
Ok, good to know there's courses without sections. I think we can go with the checkbox and chevron button hidden then, but do you think the `X` button could be centred when there's no chevron button? Probably just need to give the `VStack` 100% height

Reply:
Is this what you are wanting when you say centred? (or do you mean stretch vertically to fit the whole card).

<img width="386" alt="Capture d’écran, le 2022-03-11 à 17 07 49" src="https://user-images.githubusercontent.com/10472448/157997406-408d8e79-65ce-4f5b-8f3a-d914e09cfb00.png">

Reply:
yes that picture is exactly it :)

Reply:
I guess there is no CRN we need to surface to the user for these kind of courses? I'm guessing its like being registered into a co-op where you can't actually register yourself into it?

Reply:
Yeah, there is no CRN, and yeah you get registered by the department.

It doesn't show up in the share function of the .ics function, but this PR isn't about those so I think it is fine for now. (Don't think we want it in those, but I can see a use case for both).